### DISCUSSION TITLE
New template with KasmVNC

### DISCUSSION THREAD

**Discussion by 0ndrec** [lima-vm/lima#2885] [Ideas]
Support if you would like to see a template from the project team 😊

**Comment by afbjorklund** [comment]
It seems like it would be simple enough to add this to the regular templates:

https://www.kasmweb.com/kasmvnc/docs/latest/install.html#debian-ubuntu-kali

So you can provide the Lima template with your software?

The VNC example in Lima is for the QEMU built-in server (host)
